cypress/e2e/seededFlows/articleFlows/commentModerationTools.spec.js

```diff
@@ -30,83 +23,84 @@ describe('Moderation Tools for Comments', () => {
 
       cy.get('@trustedUser').then((trustedUser) => {
         cy.loginAndVisit(trustedUser, '/series_user/series-test-article-slug');
+        // Reference: https://filiphric.com/how-to-wait-for-page-to-load-in-cypress
+        cy.intercept('/notifications/counts').as('notificationCount');
+        cy.intercept('/async_info/base_data').as('baseData');
+        cy.intercept('/ahoy/visits').as('ahoy');
+        cy.get('#comments').within(() => {
+          cy.findByRole('button', { name: 'Toggle dropdown menu' }).click();
+          cy.findByRole('link', { name: 'Moderate' }).click();
+        });
+        cy.wait(['@notificationCount', '@baseData', '@ahoy'], {
+          timeout: 10000,
+        });
```

Comment:
```suggestion
        cy.wait(['@notificationCount', '@baseData', '@ahoy']);
```
I rather use the default wait time than explicitly making it longer. If in the real world scenario, our users need to wait that long for the page to load, I rather we find out from test failure.